test/modules/TwinTester/TwinEdgeOperationsInitializer.cs

```diff
@@ -36,7 +36,6 @@ public static async Task<TwinEdgeOperationsInitializer> CreateAsync(RegistryMana
 
                 // reset reported properties
                 await TwinTesterUtil.ResetTwinReportedPropertiesAsync(moduleClient, twin);
-                
```

Comment:
Nit: Revert this file's change? It just pollutes the git history.

Reply:
+1